sweep/tx_input_set.go

```diff
@@ -83,14 +83,18 @@ type txInputSet struct {
 	wallet Wallet
 }
 
+func dustLimit(relayFee chainfee.SatPerKWeight) btcutil.Amount {
+	return txrules.GetDustThreshold(
+		input.P2WPKHSize,
```

Comment:
This shouldn't always be `P2WKH` since at times we spend `P2WSH` outputs which will typically have a higher dust limit (since the total input size to spend them is typically larger). We also have this open issue tracking this matter elsewhere in the codebase: https://github.com/lightningnetwork/lnd/issues/3946

Reply:
Yeah, this was pre-existing. Makes sense to fix separately IMO.